Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/jupyter_cache/executors/utils.py", line 56, in single_nb_execution
    record_timing=False,
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/asyncio/base_events.py", line 587, in run_until_complete
    return future.result()
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 559, in async_execute
    cell, index, execution_count=self.code_cells_executed + 1
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
from itables import show
show()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mAttributeError[0m                            Traceback (most recent call last)
[0;32m/tmp/ipykernel_2570/3250159183.py[0m in [0;36m<module>[0;34m[0m
[1;32m      1[0m [0;32mfrom[0m [0mitables[0m [0;32mimport[0m [0mshow[0m[0;34m[0m[0;34m[0m[0m
[0;32m----> 2[0;31m [0mshow[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m
[0;32m/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/itables/javascript.py[0m in [0;36mshow[0;34m(df, **kwargs)[0m
[1;32m    170[0m [0;32mdef[0m [0mshow[0m[0;34m([0m[0mdf[0m[0;34m=[0m[0;32mNone[0m[0;34m,[0m [0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    171[0m     [0;34m"""Show a dataframe"""[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 172[0;31m     [0mhtml[0m [0;34m=[0m [0m_datatables_repr_[0m[0;34m([0m[0mdf[0m[0;34m,[0m [0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    173[0m     [0mload_datatables[0m[0;34m([0m[0mskip_if_already_loaded[0m[0;34m=[0m[0;32mTrue[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    174[0m     [0mdisplay[0m[0;34m([0m[0mHTML[0m[0;34m([0m[0mhtml[0m[0;34m)[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/itables/javascript.py[0m in [0;36m_datatables_repr_[0;34m(df, tableId, **kwargs)[0m
[1;32m    107[0m         [0mdf[0m [0;34m=[0m [0mdf[0m[0;34m.[0m[0mto_frame[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    108[0m [0;34m[0m[0m
[0;32m--> 109[0;31m     [0mdf[0m [0;34m=[0m [0mdownsample[0m[0;34m([0m[0mdf[0m[0;34m,[0m [0mmax_rows[0m[0;34m=[0m[0mmaxRows[0m[0;34m,[0m [0mmax_columns[0m[0;34m=[0m[0mmaxColumns[0m[0;34m,[0m [0mmax_bytes[0m[0;34m=[0m[0mmaxBytes[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    110[0m [0;34m[0m[0m
[1;32m    111[0m     [0;31m# Do not show the page menu when the table has fewer rows than min length menu[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;32m/opt/hostedtoolcache/Python/3.7.12/x64/lib/python3.7/site-packages/itables/downsample.py[0m in [0;36mdownsample[0;34m(df, max_rows, max_columns, max_bytes)[0m
[1;32m      9[0m [0;32mdef[0m [0mdownsample[0m[0;34m([0m[0mdf[0m[0;34m,[0m [0mmax_rows[0m[0;34m=[0m[0;36m0[0m[0;34m,[0m [0mmax_columns[0m[0;34m=[0m[0;36m0[0m[0;34m,[0m [0mmax_bytes[0m[0;34m=[0m[0;36m0[0m[0;34m)[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m     10[0m     [0;34m"""Return a subset of the dataframe that fits the limits"""[0m[0;34m[0m[0;34m[0m[0m
[0;32m---> 11[0;31m     [0morg_rows[0m[0;34m,[0m [0morg_columns[0m[0;34m,[0m [0morg_bytes[0m [0;34m=[0m [0mlen[0m[0;34m([0m[0mdf[0m[0;34m.[0m[0mindex[0m[0;34m)[0m[0;34m,[0m [0mlen[0m[0;34m([0m[0mdf[0m[0;34m.[0m[0mcolumns[0m[0;34m)[0m[0;34m,[0m [0mdf[0m[0;34m.[0m[0mvalues[0m[0;34m.[0m[0mnbytes[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m     12[0m     df = _downsample(
[1;32m     13[0m         [0mdf[0m[0;34m,[0m [0mmax_rows[0m[0;34m=[0m[0mmax_rows[0m[0;34m,[0m [0mmax_columns[0m[0;34m=[0m[0mmax_columns[0m[0;34m,[0m [0mmax_bytes[0m[0;34m=[0m[0mmax_bytes[0m[0;34m[0m[0;34m[0m[0m

[0;31mAttributeError[0m: 'NoneType' object has no attribute 'index'
AttributeError: 'NoneType' object has no attribute 'index'

